test/WebJobs.Script.Tests.Integration/TestScripts/Node/Scenarios/index.js

```diff
@@ -88,8 +87,8 @@ var assert = require('assert');
     else if (scenario === 'appInsights') {
 
         var logPayload = {
-            InvocationId: context.executionContext.invocationId,
-            Trace: input.value
+            invocationId: context.executionContext.invocationId,
+            trace: input.value
         };
 
         context.log(JSON.stringify(logPayload));
```

Comment:
out of curiosity, does this pass if you don't stringify the log payload? iirc we're doing that under the covers

Reply:
Looks like they do pass. Not sure why I did that :-)